### DISCUSSION TITLE
Generate more rows than TotalRows

### DISCUSSION THREAD

**Discussion by IlyaZh** [qax-os/excelize#1781] [General]
Why is amount of rows is hard limited?
Is it possible to override this value?

I would like to generate more than 1kk rows on my own risks.

**Comment by xuri** [comment]
This similar with the issue #1101, Excel has maximum row limits (1,048,576 rows), please reference [Excel specifications and limits](https://support.microsoft.com/en-us/office/excel-specifications-and-limits-1672b34d-7043-467e-8e27-269d656771c3).

**Reply by IlyaZh** [reply]
But it still works with a bigger amount of rows)

**Reply by xuri** [reply]
If we write rows over the maximum row limits, Excel will prompt the file corrupted when opening the generated workbook, so we need to keep this limit guard.
